src/cli.ts

```
@@ -0,0 +1,156 @@
+  // Use parseAsync to correctly handle asynchronous command actions
+  await program.parseAsync(process.argv);
+
+  // If no commands or arguments are provided, show help
+  if (!process.argv.slice(2).length) {
+    program.outputHelp();
+  }
+}
+
+// Check if the script is run directly (as a CLI)
+if (require.main === module) {
+  main();
```

ESLint

[error] Extra semicolon.

(style/semi)